### DISCUSSION TITLE
Flit style labeling for flash

### DISCUSSION THREAD

**Discussion by BSIT-Evanism** [LazyVim/LazyVim#1122] [Q&A] [answered]
Hi, I really love the flit using the f-t motion navigating to my codes, with the recent change, I've really weirded out with the new blocky labels compared to subtle blue highlights on flit, any ways so I bring back the style in flash.nvim plugin?

**Comment by folke** [comment, marked as answer]
Use this to get something similar to flit:

```lua
{
    "folke/flash.nvim",
    opts = {
      modes = {
        char = {
          highlight = {
            groups = {
              label = "LeapLabelPrimary",
            },
            matches = false,
          },
        },
      },
    },
  }

**Reply by BSIT-Evanism** [reply]
Thanks for the solution mate, really appreciate it
